create.ps1

```diff
@@ -163,136 +108,166 @@ function Resolve-ZivverError {
         Write-Output $httpErrorObj
     }
 }
+#endregion functions
 
-#endregion
-
-# Begin
 try {
-    Write-Verbose "Verify if [$account.userName] has a value"
-    if ([string]::IsNullOrEmpty($($account.userName))) {
-        throw 'Mandatory attribute [$account.userName] is empty.'
+    #region Verify correlation configuration and properties
+    $actionMessage = "verifying correlation configuration and properties"
+
+    if ($actionContext.CorrelationConfiguration.Enabled -eq $true) {
```

Comment:
Ik zie deze manier van bracing vaak terugkomen. 
```
if () {
}
else {
}
```

Wij (CT) doen dit meestal net even anders. Heeft de manier zoals hierboven een beetje jullie voorkeur? Anders neem ik dit even mee om te bespreken in het CT.

Reply:
CTRL + SHIFT + F doet dit standaard in VScode. Dat is eigenlijk de enige afspraak die we binnen consultancy hebben.